style.css

```diff
@@ -0,0 +1,137 @@
+body {
+  background-color: black;
+}
+
+.container {
+  display: flex;
+  justify-content: center;
+  background-color: black;
+  height: 2000px;
+}
+
+.mobile-view {
+  display: grid;
+  grid-template-columns: 400px;
+  justify-content: center;
+  background-color: #1c1a19;
+}
+
+.mobile-nav {
+  height: 92px;
+  background-color: #3c3a39;
+  position: fixed;
+  right: 560px;
+  top: 0;
+  left: 560px;
+}
```

Comment:
- [ ] Kindly make sure your hamburger icon is showing on mobile devices as requested by the previous reviewer.
Currently, it is not showing up on mobile devices.

<img src="https://user-images.githubusercontent.com/17243325/205438108-fa4ecdfb-ca0d-4afb-8085-f8c3aefda26b.png" width="300">